### PR TITLE
[guides] add guide for Updates QA

### PR DESCRIPTION
**Pull request by esamelson**
# Why

I did a full QA of updates for SDK 41, since we made a lot of changes during the SDK cycle related to EAS update. Wanted to add a guide for this in case we want to do it again.

# How

Wrote down all the steps as I went through the full QA process.

### CODE REVIEW

File: guides/releasing/Updates QA.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Updates QA
+
+## Expo Go (iOS and Android)
+
+- Publish an app with the new SDK version and `fallbackToCacheTimeout: 0`. Open it in Expo Go.
```
Comment (ide) [inline guides/releasing/Updates QA.md:5]:
```suggestion
- Publish a classic update with the new SDK version and `fallbackToCacheTimeout: 0`. Open it in Expo Go.
```

File: guides/releasing/Updates QA.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+- Turn wifi off, close and restart Expo Go, open the same project; should see the most recent update along with the "cached update" alert.
+- Add `developmentClient: { silentLaunch: true }` to app.json and publish a new update.
+- Close and restart Expo Go, open the project again. The "New update available, downloading" bar should NOT show on the splash screen this time.
+- Make a change to the UI and manifest, publish to the `staging` release channel.
+- Open the app from the `staging` channel in Expo Go; ensure you see the correct splash screen & UI. Try switching back and forth between the channels.
+- Try loading an older project with an unsupported SDK version; should see a helpful error message about SDK versions.
+- Add some buttons to your UI that test the `Updates` module methods and publish a new update.
+- Test `checkForUpdateAsync`, `fetchUpdateAsync`, and `reloadAsync`; publish another new update and use these methods to download the update and reload with it.
+- If any changes were made to the ABIXX updates files, test the module methods in all supported SDK versions.
```
Comment (ide) [inline guides/releasing/Updates QA.md:16]:
```suggestion
- If any changes were made to the ABIXX `expo-updates` files, test the module methods in all supported SDK versions.
```

File: guides/releasing/Updates QA.md [hunk 3]
```diff
@@ -0,0 +1,81 @@
+# Updates QA
+
+## Expo Go (iOS and Android)
+
+- Publish an app with the new SDK version and `fallbackToCacheTimeout: 0`. Open it in Expo Go.
+- Publish an update with changes to the UI and manifest (splash background color is easy).
+- Close and restart Expo Go. Open the project again, should see the changes to the splash screen and app UI right away (ignoring `fallbackToCacheTimeout`).
+- Turn wifi off, close and restart Expo Go, open the same project; should see the most recent update along with the "cached update" alert.
```
Comment (ide) [inline guides/releasing/Updates QA.md:8]: Should this say to turn off all network access or just Wi-Fi?

File: guides/releasing/Updates QA.md [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  - Uninstall the old build
+  - Turn off wifi on your emulator/device.
+  - Install and launch the new build, ensure the embedded update loads immediately.
+  - Turn wifi back on.
+  - Publish a new update.
+  - Relaunch, see the embedded update (new update should be downloading in the background).
+  - Close app after a few seconds and turn wifi off.
+  - Relaunch, see new update.
+- Publish another new update and make a new build. This time, **don't uninstall** the old app -- install the new build on top of the old one.
+- Make sure wifi is still off, and then launch the app; you should see the newest (embedded) update immediately. Turn on wifi and relaunch teh app a couple more times to ensure the new update persists.
```
Comment (ide) [inline guides/releasing/Updates QA.md:47]:
```suggestion
- Make sure wifi is still off, and then launch the app; you should see the newest (embedded) update immediately. Turn on wifi and relaunch the app a couple more times to ensure the new update persists.
```

File: guides/releasing/Updates QA.md (hunk 3 above)
Comment (esamelson) [reply guides/releasing/Updates QA.md]: good point -- revised all these to say "disable / reenable network access"

**Review by ide — APPROVED** [review]

**Pull request merged**